ASB SEP 2025 Security Patches integration

AlamIntel opened the pull request:
Integrating Google Android Security Bulletin Patches

Test done: STS r42 TCs Passed except 14 Test.
Screen lock issue (Issue Id: 134026)
android.security.sts.KernelLtsTest#testRequiredKernelLts_WARN
(Issue ID: 133684)

Tracked-On: OAM-133870

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

AlamIntel committed:
ASB SEP 2025 Security Patches integration

Integrating Google Android Security Bulletin Patches

Test done: STS r42 TCs Passed except 14 Test.
Screen lock issue (Issue Id: 134026)
android.security.sts.KernelLtsTest#testRequiredKernelLts_WARN
(Issue ID: 133684)

Tracked-On: OAM-133870
Signed-off-by: Alam, SahibeX <sahibex.alam@intel.com>

sysopenci commented:
FAILURE: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

tanujtekriwal reviewed (commented):
LGTM

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

tanujtekriwal reviewed (approved):
LGTM

shyjumon-n reviewed (approved):
LGTM

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /android-ci-local/celadon_umr0_master-merge/514
